src/main/java/com/odysseusinc/arachne/datanode/scheduler/CentralScheduler.java

```diff
@@ -77,12 +76,18 @@ private void checkModeSwitching() {
 
         List<User> users = userService.findStandaloneUsers();
         if (users.size() > 0) {
-            throw new BeanInitializationException(MessageFormat.format(MODE_ERROR_MESSAGE, "users"));
+            throw new BeanInitializationException("Cannot switch mode from Standalone to Network - there are some users not linked to the Central.");
```

Comment:
What is the objective of this change?

Reply:
I added a third error message that is no longer matches the constant template. And it turns out that in two cases we use a constant for an error and in the third just a string. I think that the approach should be the same everywhere.